shotover-proxy/src/transforms/cassandra/sink_cluster/token_map.rs

```diff
@@ -30,6 +30,14 @@ impl TokenMap {
             .take(replica_count)
             .map(|(_, node)| *node)
     }
+
+    // pub fn iter_replica_nodes(&self, token: Murmur3Token) -> impl Iterator<Item = Uuid> + '_ {
```

Comment:
commented code

Reply:
Yeah I was just leaving that around in case we needed it but I'll remove it.